source/sass/base.scss

```diff
@@ -1,4 +1,6 @@
-@import "fonts";
+$fonts-path: "../fonts";
```

Comment:
It's not really config yet, since the assets will rely on the same `export` structure. I would like to make it configurable, but relies on coordinating references to all assets. This just avoids have to rewrite in Gulp, or have incorrect paths in the generated Sass.

Reply:
Fair enough, works for me for now.